components/core/View.test.tsx

```diff
@@ -25,39 +26,39 @@ describe('View', () => {
   });
```

Comment:
The test above this shows an edge-case for how we have these factored right now - colors not given as top-level attributes don't get put through `colorLookup`. Getting stuff like this ironed out is likely harder than just using Tamagui.

Reply:
Yes, I think the way they do Styled components does entirely make this error case impossible.